test/features/deploy_quarkus_service.feature

```diff
@@ -118,3 +118,39 @@ Feature: Deploy quarkus service
     Examples: Native
       | native  | minutes |
       | enabled | 20      |
+
+#####
+
+  @events
+  @persistence
+  Scenario Outline: Deploy Quarkus process instance with events stored in Data Index
```

Comment:
now make me think we could improve the name :D

something like `Retrieve Quarkus Process Instance events from Data Index`

wdyt ? make sense ?
@sutaakar @MarianMacik also ?

Reply:
Hehe, I do agree with you. What about this name: "Deploy process-quarkus-example service with cloud events" ?

Reply:
nope I think the scenario's name should show what the `Then` steps do

Reply:
and maybe the `When` if not enough explicit

Reply:
As per @radtriste suggested, I changed the title to "Data Index retrieves Spring Boot process' events"